pom.xml

```diff
@@ -88,20 +90,21 @@
     <version.equalsverifier>3.15.2</version.equalsverifier>
     <version.openpojo>0.9.1</version.openpojo>
     <version.jacoco>0.8.11</version.jacoco>
-    <version.slf4j-test>1.2.0</version.slf4j-test>
+    <version.slf4j-test>3.0.1</version.slf4j-test>
     <version.auto-restdocs>2.0.11</version.auto-restdocs>
     <version.testcontainers>1.19.0</version.testcontainers>
+    <version.mockito-inline>5.2.0</version.mockito-inline>
 
     <!-- AspectJ dependencies -->
     <version.aspectj-maven-plugin>1.14.0</version.aspectj-maven-plugin>
     <version.aspectj>1.9.20.1</version.aspectj>
 
     <!-- Excel to DMN converter dependencies -->
     <version.dmn-xlsx-converter>0.3.0</version.dmn-xlsx-converter>
-    <version.jaxb-api>2.3.1</version.jaxb-api>
-    <version.jaxb-core>4.0.4</version.jaxb-core>
+    <version.jakarta.xml.bind-api>2.3.3</version.jakarta.xml.bind-api>
+    <version.jaxb-core>2.3.0.1</version.jaxb-core>
```

Comment:
Hallo @oltoko ,
yes at first glance it looks like a downgrade. :)

Therefore, here is a technical explanation of what happened:

It is actually about exactly this block in the parent pom
```xml
<!-- this block from master -->
    <version.jaxb-api>2.3.1</version.jaxb-api>
    <version.jaxb-core>4.0.4</version.jaxb-core>
    <version.jaxb-impl>2.3.8</version.jaxb-impl>
    <version.javax.activation>1.1.1</version.javax.activation>
<!-- became this block -->
    <version.jakarta.xml.bind-api>4.0.1</version.jakarta.xml.bind-api>
    <version.glassfish-jaxb-core>4.0.3</version.glassfish-jaxb-core>
    <version.glassfish-jaxb-runtime>4.0.3</version.glassfish-jaxb-runtime>
    <version.jakarta.activation>2.1.2</version.jakarta.activation>
```

But you have to look at each line individually and where the variables are used.

`<version.jaxb-core>4.0.4</version.jaxb-core>` became `<version.glassfish-jaxb-core>4.0.3</version.glassfish-jaxb-core>`

The property is used in `taskana-routing-rest`

```xml
<!-- This one -->
    <dependency>
      <groupId>com.sun.xml.bind</groupId>
      <artifactId>jaxb-core</artifactId>
      <version>${version.jaxb-core}</version>
    </dependency>
<!-- became this one -->
    <dependency>
      <groupId>org.glassfish.jaxb</groupId>
      <artifactId>jaxb-core</artifactId>
      <version>${version.glassfish-jaxb-core}</version>
    </dependency>
```

`<version.jaxb-api>2.3.1</version.jaxb-api>` became `<version.jakarta.xml.bind-api>4.0.1</version.jakarta.xml.bind-api>`

The property is used in `taskana-routing-rest`

```xml
<!-- This one -->
    <dependency>
      <groupId>javax.xml.bind</groupId>
      <artifactId>jaxb-api</artifactId>
      <version>${version.jaxb-api}</version>
    </dependency>
<!-- became this one -->
    <dependency>
      <groupId>jakarta.xml.bind</groupId>
      <artifactId>jakarta.xml.bind-api</artifactId>
      <version>${version.jakarta.xml.bind-api}</version>
    </dependency>
```


`<version.jaxb-impl>2.3.8</version.jaxb-impl>` became `<version.glassfish-jaxb-runtime>4.0.3</version.glassfish-jaxb-runtime>`

The property is used in `taskana-routing-rest`

```xml
<!-- This one -->
    <dependency>
      <groupId>com.sun.xml.bind</groupId>
      <artifactId>jaxb-impl</artifactId>
      <version>${version.jaxb-impl}</version>
    </dependency>
<!-- became this one -->
    <dependency>
      <groupId>org.glassfish.jaxb</groupId>
      <artifactId>jaxb-runtime</artifactId>
      <version>${version.glassfish-jaxb-runtime}</version>
    </dependency>
```

`<version.javax.activation>1.1.1</version.javax.activation>` became `<version.jakarta.activation>2.1.2</version.jakarta.activation>`

The property is used in `taskana-routing-rest`

```xml
<!-- This one -->
    <dependency>
      <groupId>javax.activation</groupId>
      <artifactId>activation</artifactId>
      <version>${version.javax.activation}</version>
    </dependency>
<!-- became this one -->
    <dependency>
      <groupId>jakarta.activation</groupId>
      <artifactId>jakarta.activation-api</artifactId>
      <version>${version.jakarta.activation}</version>
    </dependency>
```

Reply:
As this line was deleted, looks fine for me now 🙂

Reply:
Hi @arolfes My initial reaction was : Why change it (sun jaxb) if we don't really have a reason to? But since you made sure it is also compatible with glassfish this is fine for me